src/components/FwbInput/FwbInput.vue

```diff
@@ -70,6 +72,7 @@ const props = withDefaults(defineProps<InputProps>(), {
   required: false,
   size: 'md',
   type: 'text',
+  autocomplete: 'off',
```

Comment:
since default value for this attribute is `'on'` I would set is as `on` in FlowbiteVue as well to avoid confusion.